schema/paramset.json

```diff
@@ -6,34 +6,48 @@
         "name": {
             "type": "string"
         },
+        "comment": {
+            "type": "string"
+        },
         "security level": {
             "type": "object",
             "properties": {
                 "classical": {
                     "type": "integer",
-                    "minimum": 1
+                    "minimum": 1,
+                    "$comment": "measured in bits"
                 },
                 "quantum": {
                     "type": "integer",
-                    "minimum": 1
+                    "minimum": 1,
+                    "$comment": "measured in bits"
                 },
                 "nist category": {
                     "type": "integer",
                     "minimum": 1,
                     "maximum": 5
+                },
+                "comment": {
+                    "type": "string"
                 }
             },
             "required": [
                 "classical",
                 "quantum"
-            ]
+            ],
+            "$comment": "#XXX Do we really always have a quantum estimation?"
```

Comment:
We should have one... otherwise having it as a pq scheme would be stupid :upside_down_face:

Reply:
OK. If we include RSA/DH for reference, we can still set this to `0`.